TACC/Core-CMS-Custom
# Task/wp 735 other exception form

## Overview

Creates post request, form validation, and form fields specific to an 'other' type of exception request/

## Related
- [WP-735](https://tacc-main.atlassian.net/browse/WP-123)

## Testing

1. Go to [http://localhost:8000/submissions/exception/](http://localhost:8000/submissions/exception/) and select Other Exception
2. Check the fields to make sure they match prod (business name placement now matches threshold form placement)
3. Submit other type exception and check that it either catches invalid fields or successfully submits
4. Check code for any logic issues or things that could be cleaned up
5. Try submitting a threshold request and make sure it still validates and submits correctly.


## UI
####Empty form
![Screenshot 2024-10-30 at 6 58 46 PM](https://github.com/user-attachments/assets/5ff8d879-d521-43d3-a3de-ef4ed5b4cebe)

#### Invalid fields
![Screenshot 2024-10-30 at 7 00 21 PM](https://github.com/user-attachments/assets/ac73d42b-c510-4666-ade6-c9c1df67b684)

#### Resets form to initial values on successful submit
![Screenshot 2024-10-30 at 7 00 55 PM](https://github.com/user-attachments/assets/744bea16-7373-4357-855c-d93bb137c5f6)

…
